ibm/service/power/resource_ibm_pi_instance_test.go

```diff
@@ -307,7 +307,7 @@ func TestAccIBMPIInstanceDeploymentType(t *testing.T) {
 		CheckDestroy: testAccCheckIBMPIInstanceDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccCheckIBMPIInstanceDeploymentTypeConfig(name, helpers.PIInstanceHealthWarning),
+				Config: testAccCheckIBMPIInstanceDeploymentTypeConfig(name, "OK", "EPIC", "e980"),
```

Comment:
Please add the resource attribute tests for these given values.

Reply:
What do you mean by `resource attribute tests` ?

Reply:
@yussufsh  I have added Variable for `OK` . We can't have default value for deployment type because they have no correlation with one another.